resources/asciidoctor/lib/elastic_include_tagged/extension.rb

```diff
@@ -28,7 +28,8 @@ def process(_doc, reader, target, attrs)
     start_match = /^(\s*).+tag::#{tag}\b/
     end_match = /end::#{tag}\b/
 
-    path, target_type, relpath = reader.resolve_include_path target, attrs, attrs
+    path, target_type, relpath =
+      reader.resolve_include_path target, attrs, attrs
```

Comment:
You might as well put each assignment on a separate line, as using parallel assignment isn't as clear when it extends to >1 lines.